pages/sidecar.rst

```diff
@@ -7,7 +7,7 @@ Graylog Sidecar
 .. note::
  Graylog 3.0 comes with a new Sidecar implementation.
  We still support the old **Collector Sidecars**, which can be found in the ``System / Collectors (legacy)`` menu entry.
- In case you need to configure legacy **Collector Sidecar** please refer to the `Graylog Collector Sidecar documentation <..2.5/collector_sidecar.html>`_.
+ In case you need to configure legacy **Collector Sidecar** please refer to the `Graylog Collector Sidecar documentation <../2.5/collector_sidecar.html>`_.
```

Comment:
I could only find one example of link to another version in the docs, and it's using the whole URL for it: https://github.com/Graylog2/documentation/blame/2.5/pages/users_and_roles/external_auth.rst#L85

I can merge if you just want to try it, though.